stdlib/builtins.pyi

```diff
@@ -1284,7 +1284,7 @@ if sys.version_info >= (3, 8):
     @overload
     def pow(base: float, exp: int, mod: None = ...) -> float: ...
     @overload
-    def pow(base: float, exp: float, mod: None = ...) -> Any: ...  # return type could be float or complex
+    def pow(base: float, exp: float, mod: None = ...) -> Any: ...  # return type could be float or complex depending on x
```

Comment:
depending on exp

Reply:
It also depends on base. If the base is positive, then any power will also be positive. Complex numbers need a negative base and a negative power.

Maybe the whole "depending on" part should be deleted? It obviously depends on the arguments in some way, which is really all that needs to be said.

Reply:
It's worth spelling out the conditions... `pow` is like catnip for typeshed contributors, because everyone always starts off thinking only about positive integers :-)

Reply:
Hah, sorry about that @hauntsaninja